.git_archival.txt

```diff
@@ -1 +1,5 @@
+node: $Format:%H$
+node-date: $Format:%cI$
+describe-name: $Format:%(describe:tags=true)$
 ref-names: $Format:%D$
```

Comment:
https://setuptools-scm.readthedocs.io/en/latest/usage/#git-archives

`ref-names` is actually considered harmful and is unnecesary now.